galaxy_ng/app/api/ui/views/sync.py

```diff
@@ -0,0 +1,62 @@
+from galaxy_ng.app.constants import COMMUNITY_DOMAINS
+from rest_framework.request import Request
+from rest_framework.response import Response
+from rest_framework.exceptions import ValidationError
+
+from django.shortcuts import get_object_or_404
+from django.utils.translation import gettext_lazy as _
+
+from pulp_container.app import models as pulp_models
+from pulp_container.app.tasks.synchronize import synchronize as container_sync
+
+from pulpcore.plugin.tasking import dispatch
+
+from galaxy_ng.app.api import base as api_base
+from galaxy_ng.app.access_control import access_policy
+from galaxy_ng.app import models
+
+
+class SyncRemoteView(api_base.APIView):
+    permission_classes = [access_policy.RemoteSyncAccessPolicy]
+    action = 'sync'
+
+    def post(self, request: Request, *args, **kwargs) -> Response:
+        distro_path = kwargs['path']
+        distro = get_object_or_404(pulp_models.ContainerDistribution, base_path=distro_path)
```

Comment:
For this bit, you should have the pk for the remote in the url, so you'd do `remote = get_object_or_404(pulp_models.ContainerDistribution, kwargs['pk']` or something like that.

Reply:
Once the remote is loaded from the database, you need to find the registry that's associated with it and copy the registry connection information into the remote.